packages/fast-jss-manager-react/README.md

```diff
@@ -218,3 +218,14 @@ const instanceStyles = {fancyButton: { color: "red" }};
 Style element order is determined by rendering order, where components higher in the React tree will have style elements defined later in the DOM (giving them higher specificity). In general, parent components will have a style element defined later in the DOM than a child component, giving the parent component greater specificity than a child.
 
 While in general this works, there is a known issue being tracked [here](https://github.com/microsoft/fast-dna/issues/1279) where parents can lose their ability to override child components. This is an unintended side-effect of memozing stylesheets. While not ideal, specificity can usually be added to the selector to mitigate the issue, as the issue only manifests when document order is determining selector specificity.
+
+### Custom JSS instance
+The JSSManager can be configured to use a custom JSS instance for all instance of the JSSManager. To ensure all components in your app use the same JSS instance to generate stylesheets, you should assign the new JSS instance prior to rendering any components.
+
+```js
+import {  create } from "jss";
+import { JSSManager } from "@microsoft/fast-jss-manager-react";
+
+
+JSSManager.jss = create(); // Assign a custom JSS instance. This instance will be used for *all* components created by the `manageJss` HOC.
```

Comment:
Do we want to link here to what this may look like? https://cssinjs.org/jss-api?v=v10.0.0-alpha.24#create-an-own-jss-instance